src/arrayMethodPop.js

```diff
@@ -5,7 +5,14 @@
  */
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
-    // write code here
+    const elem = this[this.length - 1];
```

Comment:
```suggestion
    const lastElement = this[this.length - 1];
```
это стоит делать после if